.goreleaser.yml

```diff
@@ -25,8 +26,8 @@ snapshot:
 
 dockers:
 - image_templates:
-  - "docker.io/appuio/image-cleanup:{{ .Tag }}"
-  - "docker.io/appuio/image-cleanup:{{ .Major }}"
+  - "docker.io/appuio/image-cleanup:v{{ .Tag }}"
```

Comment:
I'd guess here the `v` is already included in `.Tag`, no?

Reply:
Or even better: let's use `.Version` **and** the `v` prefix. That way it will always be with a `v`, even if the tag doesn't contain it.

Reply:
```suggestion
  - "docker.io/appuio/image-cleanup:v{{ .Version }}"
```